docs/policies/deployment-redundancy.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: Deployment redundancy
+description: ""
+permalink: /deployment-redundancy
+parent: Writing policies
+nav_order: 4
+---
+
+# Deployment redundancy
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+## Scenario
+
+Let's say you have a deployment (`mydeployment`), fronted by a service (`myservice`). The deployment has 3 instances, and in theory, you should be able to lose one with no traffic down.
+
+Let's test that. But only during the week, and before lunch, just in case something does break.
+
+## Policy
+
+
+```yaml
+scenarios:
+- name: Deployment redundancy
+  steps:
+  - podAction:
+      matches:
+        - deployment:
+            name: mydeployment
+            namespace: example
+      filters:
+        # to restrict the actions to work days, you can do
+        - dayTime:
+            onlyDays:
+              - monday
+              - tuesday
+              - wednesday
+              - thursday
+              - friday
+            startTime:
+              hour: 10
+              minute: 0
+              second: 0
+            endTime:
+              hour: 12
+              minute: 30
+              second: 0
+
+        - randomSample:
+            size: 1
```

Comment:
But if one pod was already down for some other reason, this will result in 2 pods being down.

A better policy would be to use:
```
- property:
    name: state
    value: Running
- randomSample:
    ratio: 0.34
```

So if there were only 2 running pods at the time this scenario was executed, the `property` filter brings the list of pods down to 2.  Then the `randomSample.ratio` brings the list of pods down to (0.34 * 2) rounded down to zero.

Doesn't completely eliminate the possibility of "co-chaos", but should help to minimize the possibility.

Reply:
Good point. Added to the doc.